Gemfile.lock

```diff
@@ -316,4 +316,4 @@ DEPENDENCIES
   yajl-ruby (~> 1.2)
 
 BUNDLED WITH
-   1.10.5
+   1.11.2
```

Comment:
This will be a problem. For reference: https://github.com/alphagov/govuk-puppet/blob/e82895dbdc1d8432c9cbcfe252a420c1bb591922/modules/govuk_rbenv/manifests/all.pp